### PR TITLE
Update Windows build instructions to refer to Command Prompt more

### PR DESCRIPTION
**Pull request by winstliu**
Despite the instructions saying that all the commands assumed the usage of Bash, it was actually the other way around - all commands used cmd syntax.  Other than clarifying that, I've also simplified/streamlined some of the sections.

There is one unfortunate "bug" introduced by these changes: whichever language package is being used by Linguist to highlight `cmd` blocks doesn't differentiate between when `dir` is being used as a command or as an argument.

``` cmd
script\build --install-dir
```

compared to

``` cmd
dir
```

/cc @damieng

### CODE REVIEW

**Comment by damieng** [comment]
The docs didn't refer to bash at all - they just used the bash syntax highlighter as the cmd one is flaky at best. Reviewing changes now.

File: docs/build-instructions/windows.md [hunk 1]
```diff
@@ -21,57 +21,47 @@ You can use either:
 Whichever version you use, ensure that:
 
  * The default installation folder is chosen so the build tools can find it
- * If using Visual Studio make sure Visual C++ support is selected/installed
- * If using Visual C++ Build Tools make sure Windows 8 SDK is selected/installed
+ * Visual C++ support is selected/installed if using Visual Studio
```
Comment (damieng) [inline docs/build-instructions/windows.md]: Personally find this order less easy to skim. You have to read to the end of the line to find out if it applies to you.

File: docs/build-instructions/windows.md [hunk 2]
```diff
@@ -21,57 +21,47 @@ You can use either:
...
+ * Windows 8 SDK is selected/installed if using Visual C++ Build Tools
  * A `git` command is in your path
- * Set the `GYP_MSVS_VERSION` environment variable to the Visual Studio/Build Tools version (`2013` or `2015`) e.g. ``[Environment]::SetEnvironmentVariable("GYP_MSVS_VERSION", "2015", "User")`` in PowerShell or set it in Windows advanced system settings control panel.
+ * The `GYP_MSVS_VERSION` environment variable is set to the corresponding Visual Studio/Build Tools version (`2013` or `2015`)
 
 ## Instructions
 
-You can run these commands using Command Prompt, PowerShell or Git Shell via [GitHub Desktop](https://desktop.github.com/). These instructions will assume the use of Bash from Git Shell - if you are using Command Prompt use a backslash instead: i.e. `script\build`.
```
Comment (damieng) [inline docs/build-instructions/windows.md]: Ah, you're right. A mention of bash there.

File: docs/build-instructions/windows.md [hunk 3]
```diff
@@ -91,7 +81,7 @@ If none of this works, do install Github Desktop and use its Git Shell as it mak
   * See the next item.
 
 * `error MSB8020: The build tools for Visual Studio 201? (Platform Toolset = 'v1?0') cannot be found.`
-  * Try setting the `GYP_MSVS_VERSION` environment variable to 2013 or 2015 depending on what version of Visual Studio you are running and then `script\clean` followed by `script\build` (re-open your command prompt or Powershell window if you set it using the GUI)
+  * Try setting the `GYP_MSVS_VERSION` environment variable to 2013 or 2015 depending on what version of Visual Studio/Build Tools is isntalled and then `script\clean` followed by `script\build` (re-open the Command Prompt if you set the variable using the GUI)
```
Comment (damieng) [inline docs/build-instructions/windows.md]: Typo - "isntalled"

File: docs/build-instructions/windows.md [hunk 4]
```diff
@@ -21,57 +21,47 @@ You can use either:
 Whichever version you use, ensure that:
 
  * The default installation folder is chosen so the build tools can find it
- * If using Visual Studio make sure Visual C++ support is selected/installed
- * If using Visual C++ Build Tools make sure Windows 8 SDK is selected/installed
+ * Visual C++ support is selected/installed if using Visual Studio
+ * Windows 8 SDK is selected/installed if using Visual C++ Build Tools
  * A `git` command is in your path
- * Set the `GYP_MSVS_VERSION` environment variable to the Visual Studio/Build Tools version (`2013` or `2015`) e.g. ``[Environment]::SetEnvironmentVariable("GYP_MSVS_VERSION", "2015", "User")`` in PowerShell or set it in Windows advanced system settings control panel.
+ * The `GYP_MSVS_VERSION` environment variable is set to the corresponding Visual Studio/Build Tools version (`2013` or `2015`)
```
Comment (damieng) [inline docs/build-instructions/windows.md]: Let's keep the command-line syntax of how to set environment variables - SETX is the equivalent in command-line. It also means you don't have to reopen you command prompt as you do with the UI.

**Comment by 50Wliu** [comment]
Going to reopen this against latest master so that I can pull in the new docs without having to deal with merge conflicts.

**Pull request closed without merging**